repository/Dockerfile

```
@@ -45,17 +45,24 @@ RUN java -jar ${DISTDIR}/bin/alfresco-mmt.jar install /tmp/amps/ ${CATALINA_HOME
 
 COPY simple_modules /tmp/simple_modules
 RUN mkdir -m 750 -p ${CATALINA_HOME}/modules/platform && \
-    if [ -f /tmp/simple_modules/jars/*.jar ]; \
-    then cp /tmp/simple_modules/jars/*.jar ${CATALINA_HOME}/modules/platform/; \
-    else echo "No simple modules found"; \
-    fi && \
+    for j in /tmp/simple_modules/jars/*.jar; do \
+      if [ -f "$j" ]; then \
+        j_name=${j##*/}; \
+        echo "Adding $j_name simple module..."; \
```

if it just for the log then inline the parsing in the echo

```suggestion
        echo "Adding ${j##*/} simple module..."; \
```